general/server/plugins/tvheadend.md

```diff
@@ -45,16 +45,25 @@ The objective of the guide is to configure the Jellyfin TVHeadend plugin to back
         * Password: *The password created previously* (for example: Jellyfin_password)
       
 > [!NOTE]
-> By default the the *TVHeadend Hostname or IP Address* section is configured by default with the hostname *localhost*, it is preferable to use the IP address *127.0.0.1* instead of *localhost*. [Reference](https://emby.media/community/index.php?/topic/55768-tv-headend-plugin-where-does-it-store-data/)
+> By default the the *TVHeadend Hostname or IP Address* section is configured by default with the hostname *localhost*, it is preferable to use the IP address *127.0.0.1* instead of *localhost*. [Reference](https://emby.media/community/index.php?/topic/55768-tv-headend-plugin-where-does-it-store-data/#entry542181)
```

Comment:
Could we just get rid of the reference? It's not really needed and it'd let us break another link with emby

Reply:
I think these are fine for now, it's the same as linking to any other forum.